sys-devel/gcc/files/gcc-8.3.0-_fix_avx2_pr69471.patch

```diff
@@ -0,0 +1,49 @@
+--- a/gcc/config/i386/i386.opt	2019/02/23 18:44:54	269163
++++ b/gcc/config/i386/i386.opt	2019/02/23 19:39:35	269164
```

Comment:
There's no way I can add a patch to an important component like gcc unless I know where its coming from.  If its coming from upstream gcc, then they should not go here but into the patchset maintained by slyfox.

Reply:
ok, so lets wait for patchset revbump.